pkg/packagekit/updates.scss

```diff
@@ -153,6 +153,20 @@ div.changelog {
   }
 }
 
+/* Update error view: same styles as of updating view */
+.update-error-main-view {
+  max-width: 60rem;
+  margin: 0 auto;
+}
+
+.error-log {
+  text-align: left;
+
+  .error-log-content {
+    margin: 0 10ex;
+  }
+}
+
 .update-log {
   text-align: center;
```

Comment:
Yeah, we have alignment in CSS here too.

Generally, we should use the PF utilities, unless it's part of a larger custom block that does multiple things.

To change this would require a separate commit and is probably best left for a separate PR. (So it's fine ignore it in this PR.)